### PR TITLE
update readme to improve the open-source look'n'feel

### PR DESCRIPTION
**Pull request by monperrus**
major refactoring of readme.

- move content around
- improve text

### CODE REVIEW

**Comment by danglotb** [comment]
I kind of disagree with the clone & compilation stuff at the top of the README.

I think it would be better to say that the users should: 
* First, get the latest release and try the provided example, _i.e._ `--example` option. 
* Second, try **DSpot** on their own project.
* Eventually, clone the repository, build **DSpot** and contribute.

What do you think?

**Comment by monperrus** [comment]
> I kind of disagree with the clone & compilation stuff at the top of the README.

This is the best practice for tools that target developers, which is the case of Dspot.

The intent of this PR is to have a README that looks as much as possible to READMEs of professional, successful open-source projects.

File: README.md [hunk 1]
```diff
@@ -1,91 +1,64 @@
...
 
-### For a more step by step documentation, see the [wiki](https://github.com/STAMP-project/dspot/wiki).
+[![Build Status](https://travis-ci.org/STAMP-project/dspot.svg?branch=master)](https://travis-ci.org/STAMP-project/dspot)[![Coverage Status](https://coveralls.io/repos/github/STAMP-project/dspot/badge.svg?branch=master)](https://coveralls.io/github/STAMP-project/dspot?branch=master)
 
-### [Riot chat room](https://riot.im/app/#/room/!vnCyWaGJbxESAkzyqh:matrix.org)
+[Riot chat room](https://riot.im/app/#/room/!vnCyWaGJbxESAkzyqh:matrix.org)
 
-### What is Dspot?
+DSpot is a tool that generates missing assertions in JUnit tests.
+DSpot take as input a Java project with an existing test suite.
```
Comment (seintur) [inline README.md]: take -> takes

File: README.md [hunk 2]
```diff
@@ -1,91 +1,64 @@
...
+As output, DSpot outputs new test cases on console. 
+DSpot supports Java project built with Maven and Gradle project (see the `--automatic-builder` option)
```
Comment (seintur) [inline README.md]: Java project built with Maven and Gradle project -> Java projects built with Maven and Gradle

File: README.md [hunk 3]
```diff
@@ -1,91 +1,70 @@
...
+# DSpot
 
-### For a more step by step documentation, see the [wiki](https://github.com/STAMP-project/dspot/wiki).
+[![Build Status](https://travis-ci.org/STAMP-project/dspot.svg?branch=master)](https://travis-ci.org/STAMP-project/dspot)[![Coverage Status](https://coveralls.io/repos/github/STAMP-project/dspot/badge.svg?branch=master)](https://coveralls.io/github/STAMP-project/dspot?branch=master)
 
-### [Riot chat room](https://riot.im/app/#/room/!vnCyWaGJbxESAkzyqh:matrix.org)
+[Riot chat room](https://riot.im/app/#/room/!vnCyWaGJbxESAkzyqh:matrix.org)
 
-### What is Dspot?
+DSpot is a tool that generates missing assertions in JUnit tests.
```
Comment (danglotb) [inline README.md]: I thin this is a shame that the input-amplification part is totally skipped, WDYT?

**Pull request merged**